lib/haml_lint/tree/tag_node.rb

```diff
@@ -180,6 +180,20 @@ def object_reference_source
       @value[:object_ref][/\A\[(.*)\]\z/, 1] if object_reference?
     end
 
+    # The attributes given to the tag parsed into a Ruby syntax tree.
+    #
+    # @return [ParsedRuby] syntax tree in the form returned by Parser gem
+    def parsed_attributes
+      HamlLint::ParsedRuby.new(HamlLint::RubyParser.new.parse(hash_attributes_source || ''))
```

Comment:
Even though the original method didn't default the string to `''`, this makes it easier to use the result and shouldn't cause any side effects that I can think of.

I ran the new HEAD version against our 26,768 lines of Haml across 981 files and it didn't cause a problem.